webapp/src/ts/services/z-score.service.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable } from '@angular/core';
 
 import { ChangesService } from '@mm-services/changes.service';
 import { DbService } from '@mm-services/db.service';
+import { isNil } from 'lodash';
```

Comment:
:+1: Good to know about `lodash-es`.  

I will also roll this back to a null/undefined check. Just don't tell @craig-landry since he was the one that got me on the `isNil` bandwagon....  :smile: